### DISCUSSION TITLE
Login with openLogin(google & twitter)  works fine in localhost but gives me error in vercel (live version).

### DISCUSSION THREAD

**Discussion by NekfarBaqir** [Web3Auth/Web3Auth#1201] [Help]
I implemented the web3Auth in my react application with the guide of custom-ui-react-app. 

I created an context file for that and put it in the very top of my app and then used the exported information and login function in my login page.

[here is my context in a code file in my github](https://gist.github.com/NekfarBaqir/6ad1c55d878c33a01d0240e8174169c2)
[here is a config file imported in the context file](https://gist.github.com/NekfarBaqir/c2d1998957fdff048ff84d0c0738b48d)

In localhost it works well but in production in vercel  when I click on the login button (login function from context gets called ) after that, in the console it shows connecting for a while then it gives me the error , I toke a picture of it from clicking till the error.

Note : the errors consoled in different area of the context, for example in login itself and in events.

[Erorr Image](https://res.cloudinary.com/baqircompany/image/upload/v1673535301/Screenshot_from_2023-01-12_19-23-21_v4swo9.png)

**Comment by Saviour1001** [comment]
The error is caused because of not specifying browser versions. Here is the way to [troubleshoot](https://web3auth.io/docs/troubleshooting/react-big-int-error) the error.

**Reply by NekfarBaqir** [reply]
thanks! its was the problem.
